packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -184,6 +185,15 @@ export interface IRepository extends IResource, notifications.INotificationRuleS
     options?: notifications.NotificationRuleOptions,
   ): notifications.INotificationRule;
 
+  /**
+   * Defines a CodeStar Notification rule which triggers when a pull request is merged.
+   */
+   notifyOnPullRequestMerged(
+    id: string,
+    target: notifications.INotificationRuleTarget,
+    options?: notifications.NotificationRuleOptions,
+  ): notifications.INotificationRule;
```

Comment:
Indentation here should be 2 spaces, but you have 1. That's why the PR build is failing:

```
@aws-cdk/aws-codecommit: /codebuild/output/src495685165/src/github.com/aws/aws-cdk/packages/@aws-cdk/aws-codecommit/lib/repository.ts
@aws-cdk/aws-codecommit:   191:1  error  Expected indentation of 2 spaces but found 3  @typescript-eslint/indent
@aws-cdk/aws-codecommit: âœ– 1 problem (1 error, 0 warnings)
```

```suggestion
   notifyOnPullRequestMerged(
     id: string,
     target: notifications.INotificationRuleTarget,
     options?: notifications.NotificationRuleOptions,
  ): notifications.INotificationRule;
```